test/integration/integration_test_fixtures.py

```diff
@@ -56,13 +56,15 @@ class IntegrationTestBase():
   This class will be refactored (https://github.com/envoyproxy/nighthawk/issues/258).
   """
 
-  def __init__(self, ip_version, server_config, backend_count=1):
+  def __init__(self, ip_version, server_config, backend_count=1, bootstrap_version_arg=None):
     """Initialize the IntegrationTestBase instance.
 
     Args:
       ip_version: a single IP mode that this instance will test: IpVersion.IPV4 or IpVersion.IPV6
       server_config: path to the server configuration
       backend_count: number of Nighthawk Test Server backends to run, to allow testing MultiTarget mode
+      bootstrap_version_arg: specify a bootstrap cli argument value for the test server binary.
```

Comment:
Can we specify the type in the comment? I am assuming this is an int, so something like:

```
bootstrap_version_arg: An int, specifies a bootstrap cli argument value for the test ....
```

And once we do that, can we also make the default argument of the same type rather than None? None is of a different type which can lead to unexpected NoneType errors.

Reply:
Changing this to `int`, but still allowing `None`:  this will end up as an optional CLI argument value, where one can indicate that the this cli argument should be omitted altogether gets expressed by passing `None`. Let me know if you are comfortable with that.

Reply:
That makes sense, thank you for the explanation.

Reply:
As discussed offline, based on the explanation if actually made more sense to default the parameter to `None` if its meaning is **unset**. Sorry about the back and forth.